### ISSUE TITLE
Stembureau proces-verbalen met stembureau- lijst- en kandidaatgegevens printbaar maken

### ISSUE THREAD

**Issue by jorisleker** [kiesraad/abacus#438]
## Achtergrond
Grote gemeenten laten de processen-verbaal voor hun stembureaus door een drukker voorbedrukken. 
Kleinere gemeenten hebben die luxe niet. Toch is prettig als elk stembureau een voorbedrukt proces-verbaal heeft, waar de gegevens uit lijst met stembureaus en verkiezingsgegevens (incl. kandidatenlijst) uit Abacus al op staan.

## Wat maken we?
- voor coordinator een plek maken waar ze een set met voorbedrukte processen-verbaal (pdf) voor alle stembureaus in hun gemeente kunnen downloaden/printen.

**Comment by praseodym** [comment]
Nog belangrijker dan de stembureaugegevens is denk ik de lijst met partijen (CSO+DSO) en de kandidatenlijst (DSO) die ook in dat proces-verbaal staat.

**Comment by jorisleker** [comment]
Yes eens! Feel free om de beschrijving van het ticket aan te vullen.

**Comment by praseodym** [comment]
> Qua 'hoeveelheid werk' even superpraktisch denken: het lijkt me een typst template voor het SB pv dat je met kandidatenlijst (eventueel per stembureau) exporteert uit je eigen Abacus installatie, of zitten er meer haken en ogen aan?

Verschil DSO/CSO is relevant, nu zijn dat twee heel verschillende modellen. Met de voorgestelde wijzigingen moet het meer op elkaar gaan lijken.

Verder natuurlijk nog een stuk backend- en frontend-code om er ook PDF's van te maken.

**Comment by jorisleker** [comment]
Uitdraaien "lege" modellen met content huidige verkiezingen is functionaliteit die OSV nu al biedt toch?
